docs/reference/content/driver/tutorials/authentication.md

```diff
@@ -51,7 +51,7 @@ In MongoDB 3.0, MongoDB changed the default authentication mechanism from [`MONG
 
 To create a credential that will authenticate using the default
 authentication mechanism regardless of server version, create a
-credential using the [`createCredential`]({{<apiref "com/mongodb/MongoCredential.html#createCredential-java.lang.String-java.lang.String-char:A-">}})
+credential using the [`createCredential`]({{<apiref "com/mongodb/MongoCredential.html#createCredential(java.lang.String,java.lang.String,char:A)">}})
```

Comment:
Any idea what the `:A` refers to?

Reply:
`char:A` refers to a character array.

Reply:
This seems different to the 4.x links there they are `char[]` rather than `char:A`. eg: http://rozza.github.io/mongo-java-driver/javadoc/com/mongodb/MongoCredential.html#createCredential(java.lang.String,java.lang.String,char[])